### ISSUE TITLE
Add  download link in release pages for the new `cockroach-sql` binary + simplify some Cloud tutorials

### ISSUE THREAD

**Issue by knz** [cockroachdb/docs#13562]
Raphael 'kena' Poss (knz) commented:

tldr: the docs should  be modified to:
- add a new download link
- simplify some Cloud tutorials/docs to recommend use the small binary instead of the large one

Context:

In https://github.com/cockroachdb/cockroach/issues/75800, we've added a new `cockroach-sql` to the published artifacts along `cockroach` and the geospatial library.

This is a smaller executable (<20MB compared to `cockroach`'s 150MB+ size) which users can download when they just want to access an existing cluster remotely.

This is part of a wider initiative originally requested by @bdarnell [here](https://github.com/cockroachdb/cockroach/issues/56058), which found recently a new purpose via @kernfeld-cockroach  with our focus to Cloud: since CC end-users are not expected to run/administrate their own cluster any more, we don't need to orient them to download the full `cockroach` binary any more.

In the longer term, we wish to provide users with a CC-specific `ccloud` CLI which *embeds* the SQL shell, obviating the need for a separate download entiely. However, we're not there yet. The new `cockroach-sql` binary provides an intermediate improvement upon the current situation.

Epic: CC-5993

Jira Issue: DOC-3283

**Comment by knz** [comment]
@ianjevans do you have recommendations about who in the docs team is best to own this?

**Comment by bdarnell** [comment]
We probably need installation instructions similar to `install-cockroachdb-*.html`. The process is simpler (no geo libraries!) but we should still give people explicit instructions. (we should also make sure that all the various installation methods work, including `brew`)

On windows, this could be a big win since `cockroach-sql` wouldn't need to recommend docker (could we remove the "experimental" banner completely or could there still be readline-related experimentalness?)

**Comment by knz** [comment]
> could there still be readline-related experimentalness

IIRC libedit is simply not linked in on windows, and we don't get line editing at all. The UX is definitely sub-par.

**Comment by knz** [comment]
personally i'd prefer a future for windows where we depend on WSL and simply ship the linux binary and provide instructions on how to run in in a WSL shell. That would be so much easier all around.

**Comment by mari-crl** [comment]
:information_source: Hello! I am a human and not at all a robot! Look at my very human username! :robot: :notes:
:thinking: Although I tried very hard to figure out what to do with this issue, more powerful human brains will need to help me. (specifically: Both Github and Jira issues were modified after exalate problems) :confounded:
:arrows_counterclockwise: **Please visit this issue's mirror at [DOC-3283](https://cockroachlabs.atlassian.net/browse/DOC-3283) and try to sync the two sides up manually.** :star2:
:white_check_mark: When you're finished, comment saying as much asn a member of Developer Infrastructure will be along to finish linking. :link:
:no_entry_sign: Note that until this is done, **this issue is not and will not be synced to Jira with Exalate**. :no_entry_sign:
:sweat_smile: Feeling lost? Don't worry about it! A member of @cockroachdb/exalate-22-cleanup-team will be along shortly to help! :+1:
:construction_worker: Developer Infrastructure members: when ready, open Exalate from the right-hand menu of the mirror issue in Jira, then choose Connect and enter this issue's URN: `cockroachdb/docs-13562`. Either way, delete this comment when you're done. :key:
:pray: Thank you for your compliance, my fellow humans! :robot: :wave:
